exasol/toolbox/sphinx/multiversion/sphinx.py

```diff
@@ -40,6 +42,37 @@
 )
 
 
+class TagFormatError(Exception):
+    """
+    Exception raised for errors in the tag format.
+
+    The exception is raised when a tag is found to be incorrectly formatted.
+    """
+
+
+class ExasolVersionTag:
```

Comment:
```suggestion
class SemanticVersionTag:
```

it is not exactly about Exasol

Reply:
Yes and no, semantic version is slightly more complex than the subset we are using (`major.minor.patch`) considering `+alpha`, `rc` etc., I also wanted to make clear that it targets the version tag format we use in/for our project(s) ("What we consider as legit").